dataverk_airflow/kubernetes_operator.py

```diff
@@ -84,11 +84,13 @@ def kubernetes_operator(
     env_vars = {
         "NLS_LANG": "NORWEGIAN_NORWAY.AL32UTF8",
         "TZ": os.getenv("TZ", "Europe/Oslo"),
-        "REQUESTS_CA_BUNDLE": CA_BUNDLE_PATH,
         "KNADA_TEAM_SECRET": os.environ["KNADA_TEAM_SECRET"],
         **extra_envs
     }
 
+    if not os.getenv("INTEGRATION_TEST"):
```

Comment:
Er ikke så fan av å legge inn test-kode i koden som shippes ut. Hvorfor kan vi ikke bare lage en dummy configmap for dette? Ev. kunne cabundle vært en del av templaten vi har for Airflow?

Reply:
Ikke fan jeg heller. Kan løses med at vi i stedet endrer global values for chartet i knorten til å mounte inn ca-bundle som ekstra volum (`workers.extraVolumes`+`workers.extraVolumeMounts`) samt setter global value for env som peker REQUESTS_CA_BUNDLE til path ca-bundle mountes på (`workers.env`).